systemd/systemd.go

```diff
@@ -1064,9 +1112,21 @@ LazyUnmount=yes
 WantedBy=multi-user.target
 `
 
-func writeMountUnitFile(snapName, revision, what, where, fstype string, options []string) (mountUnitName string, err error) {
-	content := fmt.Sprintf(mountUnitTemplate, snapName, revision, what, where, fstype, strings.Join(options, ","))
-	mu := MountUnitPath(where)
+const (
+	snappyCreatorModule = "X-SnapdOrigin"
```

Comment:
nitpick: if the setting we put into the mount unit file is Origin, can we use Origin instead of Creator for all the properties/variables/etc. here too?

Reply:
Done!